todos.go

```diff
@@ -0,0 +1,109 @@
+package gitlab
+
+import "time"
+import "fmt"
+
+// TodosService handles communication with the todos related methods of
+// the Gitlab API.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/8-16-stable/doc/api/todos.md
```

Comment:
I don't know if there are any differences, but it would be much appreciated if you could use the api V4 docs and target this PR against the `f-api-v4` branch instead.